.github/pr-custom-review.yml

```diff
@@ -0,0 +1,8 @@
+rules:
+  - name: FILES
```

Comment:
Use a more precise and readable name such as "Changed runtime files"

Reply:
Fixed